file-avail-largedata.md

```diff
@@ -1,299 +1,603 @@
 ---
 highlighter: none
 #layout: default
-layout: file_avail
+layout: default
 title: Managing Large Data in HTC Jobs
 ---
 
-When submitting jobs to CHTC's High Throughput Computing (HTC) system, 
-there is a distinct location for staging data that is too large to be 
-handled at scale via the default HTCondor file transfer mechanism. This 
-location should be used for jobs that require input files larger than 100MB
-and/or that generate output files larger than 3-4GB. 
+# Data Transfer Solutions By File Size
+
+![CHTC File Management Solutions](/images/chtc-file-transfer.png)
+
+Due to the distributed nature of CHTC's High Throughput Computing (HTC) system, 
+your jobs will run on a server (aka an execute server) that is separate and 
+distinct from the server that your jobs are submitted from (aka the submit server). 
+This means that a copy of all the files needed to start your jobs must be 
+made available on the execute server. Likewise, any output files created 
+during the execution of your jobs, which are written to the execute server, 
+will also need to be transferred to an acessible location after your jobs complete. 
+**How input files are copied to the execute server and how output files are 
+copied back will depend on the size of these files.**
+
+CHTC's data filesystem called "Staging" is a distinct location for 
+temporarily hosting files that are too large to be handled in a 
+high-throughput fashion via the default HTCondor file transfer 
+mechanism which is otherwise used for small files hosted in your `/home` 
+directory on your submit server. 
+
+CHTC's `/staging` location is specifically intended for:
+
+- any individual input files >100MB   
+- input files totaling >500MB per job
+- individual output files >4GB
+- output files totaling >4GB per job 
+
+# Table of Content
+
+- [Who Should Use Staging](#use)    
+- [Policies and User Responsibilities](#policies-and-user-responsibilities)      
+- [Quickstart Instructions](#quickstart-instructions)
+- [Get Access To Staging](#access)       
+- [Use The Transfer Server To Move Files To/From Staging](#transfer)       
+- [Submit Jobs With Input Files in Staging](#input)     
+- [Submit Jobs That Transfer Output Files To Staging](#output)
+- [Tips For Success When Using Staging](#tips)
+- [Managing Staging Data and Quotas](#quota)    
+
+<a name="use"></a>
+# Who Should Use `/staging`
+
+`/staging` is a location specifically for hosting singularly larger input (>100MB) 
```

Comment:
I'm not sure about "singularly" here, but don't have any good suggestions. "Individual" maybe?

Reply:
I got this wording from @lmichael107